File: c/fib.c
#include <stdlib.h>
#include <stdio.h>
#include <time.h>

unsigned long long int fibonnaci(int n) {
  if (n <= 1) {
    return n;
  }
  return fibonnaci(n - 1) + fibonnaci(n - 2);
}

int main(int argc, char *argv[]) {
  int sequences = atoi(argv[1]);
  float start = clock();
  unsigned long long int output = fibonnaci(sequences);
  float end = clock();
  float elapsed = (end - start) / (CLOCKS_PER_SEC / 1000);
  printf("[C] Fib %dx | Result: %llu | Time: %.3fms\n", sequences, output, elapsed);
  return 0;
}

File: python/fib.py
import sys
import time

def fibonnaci(n):
  if (n <= 1):
    return n
  else:
    return fibonnaci(n - 1) + fibonnaci(n - 2)

try:
  sequences = int(sys.argv[1])
  start = time.time()
  output = fibonnaci(sequences)
  finish = time.time()
  elapsed = (finish - start) * 1000
  print("Python Fib: %dx\nResult: %d | Time: %.0fms" %(sequences, output, elapsed))
except Exception as err:
  print(err)